provisioning/local/aws.tf

```diff
@@ -0,0 +1,88 @@
+provider "aws" {
+  region  = local.region
+  profile = var.aws_profile
+}
+
+locals {
+  region = "eu-central-1"
+}
```

Comment:
Ten region nebudes nikdy menit, takze bych ho dal napevno do toho provideru a pak si hodil `data "aws_region" "current" {}` abych si ho mohl referencovat kdekoliv dal https://github.com/keboola/sandboxes-service/blob/8406d9dc004904dd374f2c6f3337f93945fe3358/provisioning/local/aws.tf#L13

Ale tohle je jen muj osobni nazor platici k dnesnimu datu, asi ne nejaka obecna best practice. Vysledek je stejny, jen to udelas zleva nebo zprava.